2019-10-21T16:16:42.7666736+01:00;VSTestExecutionThread;Info;SpecFlow+Runner execution started
2019-10-21T16:16:42.7666736+01:00;VSTestExecutionThread;Info;SpecFlow+Runner 3.0.391 in Framework clr40 in x86 mode execution started
2019-10-21T16:16:42.7666736+01:00;VSTestExecutionThread;Info;TestAdapter Location: C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\packages\SpecRun.Runner.3.0.391\tools\net45\TechTalk.SpecRun.VisualStudio.TestAdapter.dll
2019-10-21T16:16:42.8636453+01:00;VSTestExecutionThread;Info;SpecRun: running tests in C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\CSharpSpecFlowSelenium\bin\Debug\CSharpSpecFlowSelenium.dll
2019-10-21T16:16:42.8656502+01:00;VSTestExecutionThread;Info;Output folder configured to "C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\TestResults" (default)
2019-10-21T16:16:42.8676493+01:00;VSTestExecutionThread;Info;Profile: 
2019-10-21T16:16:42.9236475+01:00;VSTestExecutionThread;Info;Log file path: C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\TestResults\Unnamed project__2019-10-21T161642.log
2019-10-21T16:16:42.9536466+01:00;VSTestExecutionThread;Info;SpecFlow+Runner Evaluation Mode: Please purchase at http://www.specflow.org/plus to remove test execution delay.
2019-10-21T16:16:42.9696478+01:00;Thread#14;Info;Checking license
2019-10-21T16:16:43.0086459+01:00;Thread#14;Info;Evaluation mode: test execution starts in 9 seconds... (this delay is removed in the registered version)
2019-10-21T16:16:52.0101456+01:00;Thread#14;Info;Evaluation delay finished.
2019-10-21T16:16:52.0261480+01:00;Thread#14;Info;License check completed
2019-10-21T16:16:52.0271494+01:00;Thread#14;Info;starting test run
2019-10-21T16:16:52.0551456+01:00;Thread#14;Info;Discovering target: Default
2019-10-21T16:16:52.0561465+01:00;Thread#14;Info;Discovering tests from assembly: C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\CSharpSpecFlowSelenium\bin\Debug\CSharpSpecFlowSelenium.dll
2019-10-21T16:16:53.1301482+01:00:Appdomain Disposed
2019-10-21T16:16:53.1301482+01:00;Thread#14;Info;Discovered tests from assembly: C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\CSharpSpecFlowSelenium\bin\Debug\CSharpSpecFlowSelenium.dll
2019-10-21T16:16:53.1311504+01:00:Test Suite Dump
2019-10-21T16:16:53.1331463+01:00:#0: Unnamed project/ - 
2019-10-21T16:16:53.1331463+01:00:#1: Default - 
2019-10-21T16:16:53.1351456+01:00:#2: CSharpSpecFlowSelenium - 
2019-10-21T16:16:53.1351456+01:00:#3: BBCLogin1 - 
2019-10-21T16:16:53.2271465+01:00:#4: Invalid password - TestAssembly:CSharpSpecFlowSelenium/Feature:BBCLogin1/Scenario:Invalid+password
2019-10-21T16:16:53.2271465+01:00:End: Test Suite Dump
2019-10-21T16:16:53.3061517+01:00:Thread#0:Scheduler: get next test for thread 0
2019-10-21T16:16:53.3111494+01:00:Thread#:Scheduler: testitem #4 from normal test queue 
2019-10-21T16:16:53.3111494+01:00:Thread#0:Scheduler: considering testitem #4 for thread 0
2019-10-21T16:16:53.3251498+01:00:Thread#0:TestItem#4:testItemMatchesCurrentTestThread:UnSpecific
2019-10-21T16:16:53.3251498+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadSpecific:False
2019-10-21T16:16:53.3251498+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadUnspecific:False
2019-10-21T16:16:53.3251498+01:00:Thread#0:Scheduler: using testitem #4 for thread 0
2019-10-21T16:16:53.3281533+01:00:Thread#0:Scheduler: Test #4 is scheduled for thread
2019-10-21T16:16:53.3301503+01:00:Thread#0:Scheduler: with testItemExecutionIndex 0
2019-10-21T16:16:53.3591463+01:00:Thread #0: creating new test thread executor for CSharpSpecFlowSelenium, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null (target: Default)
2019-10-21T16:16:53.3841496+01:00:Thread #0: Creating test assembly executor for execution model AppDomain
2019-10-21T16:16:53.4021447+01:00:Thread #0: Creating test assembly executor for execution model AppDomain
2019-10-21T16:16:53.4051475+01:00:Thread #0: using TechTalk.SpecRun.Framework.Execution.InProcess.InProcAppDomainTestAssemblyExecutorHost to execute tests
2019-10-21T16:16:53.7271469+01:00:Thread#0: Test #4/0 is started on thread #0
2019-10-21T16:16:54.0071466+01:00:Thread#0:T:-> Loading plugin C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\CSharpSpecFlowSelenium\bin\Debug\SpecRun.Runtime.SpecFlowPlugin.dll
2019-10-21T16:16:54.0231462+01:00:Thread#0:T:-> Using default config
2019-10-21T16:17:01.8143755+01:00:Thread#0:B:Given I am on the login page
2019-10-21T16:17:01.8943739+01:00:Thread#0:T:error: Object reference not set to an instance of an object.
2019-10-21T16:17:01.9003745+01:00:Thread#0:B:And I have enter a valid username
2019-10-21T16:17:01.9013727+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:17:01.9013727+01:00:Thread#0:B:And I have entered a invalid password
2019-10-21T16:17:01.9033735+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:17:01.9043744+01:00:Thread#0:B:When I press login
2019-10-21T16:17:01.9043744+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:17:01.9053770+01:00:Thread#0:B:Then I should see the appropriate error
2019-10-21T16:17:01.9063753+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:17:02.0723736+01:00:Thread#0:TestItem#4:Scheduler: reschedule tests for testitem #4
2019-10-21T16:17:02.0753763+01:00:Thread#0:Scheduler: get next test for thread 0
2019-10-21T16:17:02.0753763+01:00:Thread#:Scheduler: dequeuing a rescheduled testitem: #4
2019-10-21T16:17:02.0753763+01:00:Thread#0:Scheduler: considering testitem #4 for thread 0
2019-10-21T16:17:02.0753763+01:00:Thread#0:TestItem#4:testItemMatchesCurrentTestThread:UnSpecific
2019-10-21T16:17:02.0753763+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadSpecific:False
2019-10-21T16:17:02.0753763+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadUnspecific:False
2019-10-21T16:17:02.0753763+01:00:Thread#0:Scheduler: using testitem #4 for thread 0
2019-10-21T16:17:02.0753763+01:00:Thread#0:Scheduler: Test #4 is scheduled for thread
2019-10-21T16:17:02.0753763+01:00:Thread#0:Scheduler: with testItemExecutionIndex 1
2019-10-21T16:17:02.6743745+01:00:Thread#0: Test #4/0 is finished on thread #0 as Failed within 8.9462249 seconds
2019-10-21T16:17:02.6773754+01:00:Thread#0: Test #4/1 is started on thread #0
2019-10-21T16:17:02.6783748+01:00:Thread#0:T:warning: The previous ScenarioContext was not disposed.
2019-10-21T16:17:09.2743759+01:00:Thread#0:B:Given I am on the login page
2019-10-21T16:17:09.2763752+01:00:Thread#0:T:error: Object reference not set to an instance of an object.
2019-10-21T16:17:09.2773743+01:00:Thread#0:B:And I have enter a valid username
2019-10-21T16:17:09.2773743+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:17:09.2773743+01:00:Thread#0:B:And I have entered a invalid password
2019-10-21T16:17:09.2773743+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:17:09.2783746+01:00:Thread#0:B:When I press login
2019-10-21T16:17:09.2783746+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:17:09.2783746+01:00:Thread#0:B:Then I should see the appropriate error
2019-10-21T16:17:09.2783746+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:17:09.2803785+01:00:Thread#0:Scheduler: get next test for thread 0
2019-10-21T16:17:09.2803785+01:00:Thread#:Scheduler: dequeuing a rescheduled testitem: #4
2019-10-21T16:17:09.2803785+01:00:Thread#0:Scheduler: considering testitem #4 for thread 0
2019-10-21T16:17:09.2803785+01:00:Thread#0:TestItem#4:testItemMatchesCurrentTestThread:UnSpecific
2019-10-21T16:17:09.2803785+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadSpecific:False
2019-10-21T16:17:09.2803785+01:00:Thread#0:TestItem#4:runsOnOneOtherThreadUnspecific:False
2019-10-21T16:17:09.2803785+01:00:Thread#0:Scheduler: using testitem #4 for thread 0
2019-10-21T16:17:09.2803785+01:00:Thread#0:Scheduler: Test #4 is scheduled for thread
2019-10-21T16:17:09.2803785+01:00:Thread#0:Scheduler: with testItemExecutionIndex 2
2019-10-21T16:17:09.8103770+01:00:Thread#0: Test #4/1 is finished on thread #0 as Failed within 7.1330016 seconds
2019-10-21T16:17:09.8113765+01:00:Thread#0: Test #4/2 is started on thread #0
2019-10-21T16:17:09.8113765+01:00:Thread#0:T:warning: The previous ScenarioContext was not disposed.
2019-10-21T16:17:16.3340223+01:00:Thread#0:B:Given I am on the login page
2019-10-21T16:17:16.3360182+01:00:Thread#0:T:error: Object reference not set to an instance of an object.
2019-10-21T16:17:16.3360182+01:00:Thread#0:B:And I have enter a valid username
2019-10-21T16:17:16.3360182+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:17:16.3370156+01:00:Thread#0:B:And I have entered a invalid password
2019-10-21T16:17:16.3370156+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:17:16.3370156+01:00:Thread#0:B:When I press login
2019-10-21T16:17:16.3370156+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:17:16.3370156+01:00:Thread#0:B:Then I should see the appropriate error
2019-10-21T16:17:16.3370156+01:00:Thread#0:T:skipped because of previous errors
2019-10-21T16:17:16.3380171+01:00:Thread#0:Scheduler: get next test for thread 0
2019-10-21T16:17:16.3380171+01:00:Thread#:Scheduler: at the end of normal test queue
2019-10-21T16:17:16.3380171+01:00:Thread#0:Scheduler: enumerator for thread is at end
2019-10-21T16:17:16.3380171+01:00:Thread#0:Scheduler: No more tests for thread
2019-10-21T16:17:18.8090156+01:00:Thread#0: Test #4/2 is finished on thread #0 as Failed within 8.9976391 seconds
2019-10-21T16:17:18.8450164+01:00:Appdomain Disposed
2019-10-21T16:17:18.8480162+01:00:Trying to cancel thread
2019-10-21T16:17:18.8480162+01:00:Thread cancelled
2019-10-21T16:17:18.8590292+01:00:Thread#-1:Scheduler: get next test for thread -1
2019-10-21T16:17:18.8590292+01:00:Thread#:Scheduler: at the end of normal test queue
2019-10-21T16:17:18.8590292+01:00:Thread#-1:Scheduler: enumerator for thread is at end
2019-10-21T16:17:18.8710166+01:00;Thread#14;Info;test run finished
2019-10-21T16:17:18.8720177+01:00;Thread#14;Info;publishing test results
2019-10-21T16:17:18.8880188+01:00;Thread#14;Info;test results published
2019-10-21T16:17:18.8890195+01:00;Thread#14;Info;generating reports
2019-10-21T16:17:19.2070161+01:00;Thread#14;Info;Starting external report generation process
2019-10-21T16:17:22.8490352+01:00;Thread#14;Info;Successfully generated reports.
2019-10-21T16:17:22.8530177+01:00;Thread#14;Info;Successfully generated reports
2019-10-21T16:17:22.8660169+01:00:Total: 1 (test executions: 3)
2019-10-21T16:17:22.8690152+01:00:Succeeded: 0
2019-10-21T16:17:22.8690152+01:00:Ignored: 0
2019-10-21T16:17:22.8700162+01:00:Pending: 0
2019-10-21T16:17:22.8710158+01:00:Skipped: 0
2019-10-21T16:17:22.8720154+01:00:Failed: 1
2019-10-21T16:17:22.8930174+01:00;VSTestExecutionThread;Info;Adding attachments to VSTest
2019-10-21T16:17:22.8940184+01:00;VSTestExecutionThread;Info;adding Test Execution Report 'C:\Users\SSengamalam\source\repos\CSharpSpecFlowSelenium\TestResults\Unnamed project__2019-10-21T161642.html'
========== End of log file ==========
